<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PackageListWindow</class>
 <widget class="QDialog" name="PackageListWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>726</width>
     <height>564</height>
    </rect>
   </property>
   <widget class="QTableWidget" name="PkgListTb">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>521</width>
      <height>391</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>60</horstretch>
      <verstretch>20</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Default">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>150</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>显示全部快递</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>601</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QDateEdit" name="date_Date"/>
        </item>
        <item>
         <widget class="QLineEdit" name="line_Search"/>
        </item>
        <item>
         <widget class="QComboBox" name="box_Status">
          <item>
           <property name="text">
            <string>待签收</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>已签收</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>待揽收</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QWidget" name="widget_3" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>查询方式：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_UsernameSender">
          <property name="text">
           <string>发件人账号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_UsernameRecver">
          <property name="text">
           <string>收件人账号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_Index">
          <property name="text">
           <string>快递单号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_Time">
          <property name="text">
           <string>时间</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_Status">
          <property name="text">
           <string>快递状态</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btn_Search">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>查找</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
